File: Models/Models/HelpModels/TaskStatusModel.cs
namespace Models
{
    public class TaskStatusModel
    {
        private int id;
        private string task;
        private string description;
        private int user_id;
        private string status;
        private string status_desc;

        public int Id
        {
            get { return id; }
            set { id = value; }
        }
        public int User_id
        {
            get { return user_id; }
            set { user_id = value; }
        }
        public string Task
        {
            get { return task; }
            set { task = value; }
        }
        public string Description
        {
            get { return description; }
            set { description = value; }
        }
        public string Status
        {
            get { return status; }
            set { status = value; }
        }
        public string Status_desc
        {
            get { return status_desc; }
            set { status_desc = value; }
        }


        public TaskStatusModel(TaskModel task, StatusModel status)
        {
            id = task.Id;
            this.task = task.Task;
            description = task.Description;
            user_id = task.User_id;
            this.status = status.Status;
            status_desc = status.Description;
        }
    }
}

File: ClientServer/client_server/src/Models/TaskStatusModel.ts
export class TaskStatusModel 
{
    public id!:number;
    public task!:string;
    public description!:string;
    public user_id!:number;
    public status!:string;
    public status_desc!:string;
    public status_id!:number;

    constructor(id:number, task:string, description:string, user_id:number, status:string, status_desc:string, status_id:number)
    {
        this.id = id;
        this.task = task;
        this.description = description;
        this.user_id = user_id;
        this.status = status;
        this.status_desc = status_desc;
        this.status_id = status_id;
    }
}